jira/client.py

```diff
@@ -3765,7 +3767,8 @@ def create_project(
             "key": key,
             "projectTypeKey": ptype,
             "projectTemplateKey": template_key,
-            "leadAccountId": assignee,
+            "lead": assignee,
+            # "leadAccountId": assignee,
```

Comment:
should this remain here? => i'd say remove the comment if it fixes it

Reply:
Yep, there's probably a bigger conversation on how we handle Jira Cloud vs Jira Datacenter/Server, but will remove these for now